kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/container/orchestration/ContainerConfiguration.java

```diff
@@ -72,24 +70,6 @@ public String getContainerName() {
         return this.containerName;
     }
 
-    /**
-     * Returns the base image for the associated container
-     *
-     * @return
-     */
```

Comment:
Since this is API, we cannot remove API methods. We need to deprecate them first and then in 1 or 2 major releases, remove them. We should also suggest, for the time being, the alternative to use instead to allow users making the switch before removal